Yosemite/Yosemite/Stores/ProductStore.swift

```diff
@@ -41,8 +41,8 @@ public class ProductStore: Store {
             resetStoredProducts(onCompletion: onCompletion)
         case .retrieveProduct(let siteID, let productID, let onCompletion):
             retrieveProduct(siteID: siteID, productID: productID, onCompletion: onCompletion)
-        case .retrieveProducts(let siteID, let productIDs, let onCompletion):
-            retrieveProducts(siteID: siteID, productIDs: productIDs, onCompletion: onCompletion)
+        case .retrieveProducts(let siteID, let productIDs, let pageNumber, let pageSize, let onCompletion):
+            retrieveProducts(siteID: siteID, productIDs: productIDs, pageNumber: pageNumber, pageSize: pageSize, onCompletion: onCompletion)
```

Comment:
It would be ideal to add some tests in `ProductStoreTests` where there is a case of the use of `pageNumber` and `pageSize`. What do you think?

Reply:
great idea! I added a subtask in https://github.com/woocommerce/woocommerce-ios/issues/2199 to add a test case separately